packages/default-storage-backend/package.json

```diff
@@ -60,6 +60,9 @@
     "test:e2e:ios": "scripts/ios_e2e.sh 'test'",
     "test:e2e:macos": "scripts/macos_e2e.sh 'test'"
   },
+  "installConfig": {
+    "hoistingLimits": "workspaces"
+  },
```

Comment:
Do we still need this?

Reply:
yeah, without this, the deps will be hoisted which somehow breaks the example app. I opted out from having to set nohoist globally to this package only. Maybe we could move example app as separate package in future